audnauseum/state_machine/looper.py

```diff
@@ -141,62 +142,35 @@ class Looper:
          'dest': 'None'},  # Not a transition
     ]
 
-    def __init__(self, volume=1, pan=0.5, loop=None):
+    def __init__(self, loop=None):
         self.machine = Machine(model=self, states=LooperStates,
                                initial=LooperStates.IDLE,
                                transitions=Looper.transitions,
                                ignore_invalid_triggers=True,
                                after_state_change=self.echo_state_change)
-
-        self.set_default_channels()
-
```

Comment:
We need this function to scan the devices defined in the OS to make sure it is capable of 2 channels.

Reply:
Added back in a subsequent commit